vscode/webviews/chat/cells/messageCell/human/editor/ToolboxButton.tsx

```diff
@@ -79,160 +79,104 @@ export const ToolboxButton: FC<ToolboxButtonProps> = memo(({ settings, api, isFi
-                                </span>
-                                <Switch
-                                    disabled={isLoading}
-                                    checked={settings.agent?.name !== undefined}
-                                    onChange={() =>
-                                        onSubmit({
-                                            ...settings,
-                                            agent: {
-                                                name: settings.agent?.name ? undefined : 'deep-cody', // TODO: update name when finalized.
-                                            },
-                                        })
-                                    }
-                                />
-                            </div>
-                        </h2>
-                        <div
-                            id="accordion-collapse-body"
-                            className="tw-p-5 tw-flex tw-flex-col tw-gap-3 tw-my-2"
-                        >
-                            <div className="tw-text-xs">
-                                <span>
-                                    Agentic chat reflects on your request and uses tools to dynamically
-                                    retrieve relevant context, improving accuracy and response quality.{' '}
-                                    <a
-                                        target="_blank"
-                                        rel="noreferrer"
-                                        href={CODY_DOCS_CAPABILITIES_URL.href} // TODO: Replace with CODY_DOCS_AGENTIC_CHAT_URL
-                                    >
-                                        Read the docs
-                                    </a>{' '}
-                                    to learn more.
-                                </span>
-                            </div>
-                            {/* Seperator */}
-                            {settings.agent?.name && !settings.shell?.error && (
-                                <div className="tw-border-b tw-border-border tw-my-2" />
-                            )}
-                            {/* Only shows the Terminal access option if client and instance supports it */}
-                            {settings.agent?.name && !settings.shell?.error && (
-                                <div>
-                                    <div
-                                        className="tw-flex tw-items-center tw-justify-between tw-w-full tw-font-medium tw-gap-3"
-                                        aria-label="terminal"
-                                    >
-                                        <span className="tw-flex tw-gap-2 tw-items-center">
-                                            <span className="tw-font-semibold tw-text-md">
-                                                Terminal access
-                                            </span>
+                                <div
```

Comment:
```suggestion

                                <div className="tw-text-xs tw-mt-2">
                                    Agentic chat reflects on your request and uses tools to dynamically
                                    retrieve relevant context, improving accuracy and response quality.{' '}
                                    <a
                                        target="_blank"
                                        rel="noreferrer"
                                        href={CODY_DOCS_CAPABILITIES_URL.href} // TODO: Replace with CODY_DOCS_AGENTIC_CHAT_URL
                                    >
                                        Read the docs
                                    </a>{' '}
                                    to learn more.
                                </div>
                                <div
```

Move the contents of the tooltip here, where it will display when the pane is open, and allow the user to click the link.